src/views/DevelopersList.vue

```diff
@@ -28,41 +29,15 @@ export default {
         LoadingScreen,
         SubHeading
     },
-    data() {
-        return {
-            developers: [],
-            isLoading: false
-        };
-    },
     created() {
-        this.fetchDevelopers();
+        this.fetchAllDevelopers();
     },
-    methods: {
-        fetchDevelopers() {
-            this.isLoading = true;
-            const developersURL =
-                "https://raw.githubusercontent.com/AngelGarcia13/DominicanWhoCodes/master/DWC.Blazor/wwwroot/data/developers.json";
-            axios
-                .get(developersURL)
-                .then(({ data: developers }) => {
-                    this.sortDevelopers(developers);
-                })
-                .catch(error => {
-                    /* eslint-disable no-console */
-                    console.error(error);
-                })
-                .finally(() => {
-                    this.isLoading = false;
-                });
-        },
-        sortDevelopers(developers) {
-            const sortedDevelopers = developers.sort((a, b) =>
-                a.name.localeCompare(b.name, "es", { ignorePunctuation: true })
-            );
-
-            this.developers = sortedDevelopers;
-        }
-    }
+    computed: mapGetters({
+        allDevelopers: "developers/getAllDevelopers",
+    }),
```

Comment:
I rather use the spread operator for this, because it lets us add more non-vuex computed values.
```suggestion
    computed: {
    	...mapGetters({
	        allDevelopers: "developers/getAllDevelopers",
	    })
    },
```